### DISCUSSION TITLE
Not possible to customize drop-shadow filter with multiple shadows?

### DISCUSSION THREAD

**Discussion by jontebol** [tailwindlabs/tailwindcss#5548] [Help] [answered]
Title says it all... ;)

Tailwind wraps whatever i put into tailwind.config with `drop-shadow()`:

```js
     extend: {
       dropShadow: {
         '3xl': '0 35px 35px rgba(0, 0, 0, 0.25)',
       }
     }
```

This makes it impossible to even recreate the default shadows provided by Tailwind:

```css
drop-shadow(0 1px 2px rgba(0, 0, 0, 0.1)) drop-shadow(0 1px 1px rgba(0, 0, 0, 0.06));
```

Because the output becomes (double wrapped in drop-shadow):

```css
drop-shadow(drop-shadow(0 1px 2px rgba(0, 0, 0, 0.1)) drop-shadow(0 1px 1px rgba(0, 0, 0, 0.06)));
```

**Comment by adamwathan** [comment, marked as answer]
Hey! You can use array syntax to add multiple shadows:

```js
dropShadow: {
  '3xl': ['0 1px 2px rgba(0, 0, 0, 0.1)', '0 1px 1px rgba(0, 0, 0, 0.06)'],
},
```

Here's a Play:

https://play.tailwindcss.com/3bCW5zo05M?file=config

The built-in drop shadows are designed to mimic the default box shadows as closely as possible already for what it's worth. I had to make a few compromises because drop shadow doesn't support the spread parameter, but they should be pretty close:

https://twitter.com/adamwathan/status/1378777927521865730

**Reply by jontebol** [reply]
Works like a charm. Thanks @adamwathan!

**Reply by nahufede** [reply]
Hi @adamwathan 

I'm ussing this code for modify the default theme:

```
var defaultTheme = require('tailwindcss/defaultTheme');

module.exports = {
    dropShadow: {
        sm: `var(--drop-shadow-xs, ${ defaultTheme.dropShadow.sm })`,
        DEFAULT: `var(--drop-shadow-sm, ${ defaultTheme.dropShadow.DEFAULT })`,
        md: `var(--drop-shadow-md, ${ defaultTheme.dropShadow.md })`,
        lg: `var(--drop-shadow-lg, ${ defaultTheme.dropShadow.lg })`,
        xl: `var(--drop-shadow-xl, ${ defaultTheme.dropShadow.xl })`,
        '2xl': `var(--drop-shadow-xxl, ${ defaultTheme.dropShadow['2xl'] })`,
        none: `var(--drop-shadow-none, ${ defaultTheme.dropShadow['none'] })`
    }
}
```

if the variable exists, it takes its information, otherwise it uses the default theme data.

The problem is that some sizes are an array:

```
dropShadow: {
      sm: '0 1px 1px rgb(0 0 0 / 0.05)',
      DEFAULT: ['0 1px 2px rgb(0 0 0 / 0.1)', '0 1px 1px rgb(0 0 0 / 0.06)'],
      md: ['0 4px 3px rgb(0 0 0 / 0.07)', '0 2px 2px rgb(0 0 0 / 0.06)'],
      lg: ['0 10px 8px rgb(0 0 0 / 0.04)', '0 4px 3px rgb(0 0 0 / 0.1)'],
      xl: ['0 20px 13px rgb(0 0 0 / 0.03)', '0 8px 5px rgb(0 0 0 / 0.08)'],
      '2xl': '0 25px 25px rgb(0 0 0 / 0.15)',
      none: '0 0 #0000',
    }
```

When I try to use, for example, the XL size, it ends up returning the following data:

`drop-shadow(var(--drop-shadow-xl, 0 20px 13px rgb(0 0 0 / 0.03), 0 8px 5px rgb(0 0 0 / 0.08)))`

When I really need it to be:

`drop-shadow(var(--drop-shadow-xl, 0 20px 13px rgb(0 0 0 / 0.03)) drop-shadow(0 8px 5px rgb(0 0 0 / 0.08)))`

How can I get the data like that? 🙏🏻

¡Thanks!
